tools/cppcheck/scripts/enhance_dashboard_metrics.py

```
@@ -0,0 +1,455 @@
+        for profile in profiles:
+            report_path = self.reports_dir / profile / "latest" / "report.json"
+            if report_path.exists():
+                try:
+                    with open(report_path) as f:
+                        data = json.load(f)
+                        comparison[profile] = {
+                            "total_issues": len(data.get("issues", [])),
+                            "files_analyzed": data.get("files_analyzed", 0),
+                            "timestamp": data.get("timestamp", "")
+                        }
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)